css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+.material-icons {
+  margin-top: 25px;
+  font-size: 35px;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-image: url(header-shapes\ mobile@2x.png);
+  background-repeat: no-repeat;
+  border-radius: 0 0 10px 0;
+}
+
+h1 {
+  font-size: 35px;
+  padding-top: 80px;
+  color: #172b4d;
+}
+
+p {
+  font-size: 16.9px;
+  color: #172b4d;
+}
+
+span {
+  color: #6070ff;
+}
+
+.icons {
+  position: relative;
+  display: inline-block;
+}
```

Comment:
- [ ] Please make sure that the icons are aligned to the center as present in the Figma template. See the images below for more clarity 💡 Also can you please fix the LinkedIn icon as well according to how it is given in the design guidelines ? 😄📌

Your design | Given design 
---| ---|
![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/daabdb41-5876-4ef2-8c09-91030944cc27) |  ![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/27cb35a3-7569-4062-9bef-9ae5a83eb816)